<launch>
  <!-- TF names-->
  <arg name="robotPoseOutput"   default="$(find handeye_calib_sphere)/robot_pose.xml" />

  <!-- End Effector Transform, the ros topic for the tf frame
       at the tip of your robot's gripper or other tool -->
  <arg name="EETF"              default="/ee_link" />

  <!-- Robot Base Transform, the ros topic for the base tf frame of your robot -->
  <arg name="baseTF"            default="/base" />

  <arg name="cameraIntrinsicInput"  default="$(find handeye_calib_sphere)/camera_intrinsic_color.xml" />

  <node pkg="handeye_calib_sphere" type="grab_data_node" name="grab_data_node" output="screen">
  <!-- handeye_calib_camodocal arg pass -->
      <param name="robotPoseOutput" type="str"  value="$(arg robotPoseOutput)" />
      <param name="cameraIntrinsicInput" type="str" value="$(arg cameraIntrinsicInput)" />
      <param name="EETF"            type="str"  value="$(arg EETF)" />
      <param name="baseTF"          type="str"  value="$(arg baseTF)" />
      <param name="useQrExtractor"  type="bool" value="true"/>
      <param name="recordingTF"     type="bool" value="false"/>
      <param name="numOfQRcodeOnSide" type="int" value="6" />
  </node>

</launch>
